css/style.css

```
@@ -41,45 +48,132 @@ section {
   transform-origin: 0 0;
   font-family: Sofia Pro;
 }
-.wrapp{
-    width: 1170px;
-    margin: auto;
-}
-.logo{
-    float: left;
-}
-.logo img{
-    width: 100%;
-    padding: 30px 70px;
-}
-.nav-area{
-    float: right;
-    list-style: none;
-    margin-top: 30px;
-    margin-right: 20px;
-}
-.nav-area li{
-    display: inline-block;
-}
-.nav-area li a {
-color: #fff;
-text-decoration: none;
-padding: 5px 20px;
-font-family: Circular Std;
-font-size: 16px;
-}
-.nav-area li a:hover{
-    background: #fff;
-    color: #333;
-}
-.text{
-    position: absolute;
-    height: 300px;
-    margin-left: 70px;
-    margin-top: 290px;
-    text-align: left;
-}
-.text h5{
+
+.hero {
+  width: 1170px;
+  margin: auto;
+}
+
+.logo {
+  float: left;
+}
+
+.logo_hero {
+  width: 100%;
+  padding: 30px 70px;
+}
+
+.nav_area {
+  margin-top: 10px;
+  margin-right: 20px;
+  float: right;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav_item {
+  position: relative;
+}
+
+.nav_list__item {
```

nu e corect folosit bem ul aici
```suggestion
.nav__item {
```